xml/networking-octavia_admin.xml

```diff
@@ -403,6 +403,164 @@ Updated port: 0b0301b9-4ee8-4fb6-a47c-2690594173f4</screen>
    </para>
   </warning>
  </section>
+ <section xml:id="octavia-admin-delete">
+   <title>Removing load balancers</title>
+   <para>
+     The following procedure demonstrates how to delete a load
+     balancer that is in the <literal>ERROR</literal>,
+     <literal>PENDING_CREATE</literal>, or
+     <literal>PENDING_DELETE</literal> state. The following commands
+     need to be executed manually to clean up the erroring instances:
```

Comment:
The following commands? Doesn't make sense since it goes straight into a procedure. Has this been overlooked from a previous version?

Reply:
Hmm, yes, that doesn't read nicely.  I'll fix it.